virttest/bootstrap.py

```diff
@@ -955,7 +956,6 @@ def bootstrap(options, interactive=False):
                 LOG.debug("Module %s loaded", module)
 
     LOG.info("")
-    LOG.info("VT-BOOTSTRAP FINISHED")
-    LOG.debug("You may take a look at the following online docs for more info:")
-    LOG.debug(" - http://avocado-vt.readthedocs.org/")
-    LOG.debug(" - http://avocado-framework.readthedocs.org/")
+    LOG.info("vt-bootstrap finished")
```

Comment:
I understand, but how about the first commit? I especially asked you to take a look at that one as it prevented incompatibility between released versions which affects users (It broke my CI) and the first commit reverted this incompatibility while suggesting different options to allow the same functionality.

Now you released this release without the fix which means everyone will suffer from this inconsistency and will have to adopt their scripts still do you think we should include the change? Maybe it'll help some people who did not update yet.